src/midi.rs

```diff
@@ -83,54 +95,82 @@ fn get_ports(midi_in: &MidiInput, print: bool) -> Result<std::vec::Vec<midir::Mi
 pub struct MidiConnection {
     port: MidiInputPort,
     port_name: String,
-    midi_in: Option<MidiInput>
+    midi_in: Option<MidiInput>,
+}
+
+fn port_prompt(ports: &Vec<MidiInputPort>, re_ask: Option<()>) -> usize {
+    let mut choice: usize = 0;
+    if let None = re_ask {
+        println!("Please choose your midi input.");
```

Comment:
```suggestion
        println!("Choose your midi input.");
```
I really like the usage of "your", makes it feel less machiney :)

Reply:
:D